docs/integrations/gitlab-server.md

```diff
@@ -0,0 +1,97 @@
+redirect_from: /git/gitlab-server/
+published: true
+---
+You can use Probo.CI with your instance of [GitLab Server](https://about.gitlab.com/install/). Just like the [GitLab Integration](/git/gitlab/ "GitLab Integration"), Probo uses the build status pop-up to display the list of build steps as defined in your `.probo.yaml` file and their statuses. The link can be found in the Overview section of the Merge Request details page.
+
+Follow these steps to get started using Probo with your instance of GitLab Server:
+
+1. <a href="#create-an-oauth-application-on-your-gitlab-server">Create an OAuth application on your GitLab Server.</a>
+2. <a href="#register-your-gitlab-server-with-proboci">Register your GitLab Server with Probo.CI.</a>
+3. <a href="#authenticate-your-gitlab-server-with-proboci">Authenticate your GitLab Server with Probo.CI.</a>
+4. <a href="#activate-a-probo-organization">Activate a Probo Organization.</a>
+5. <a href="#activating-repositories">Activate repositories.</a>
+
+## Create an OAuth application on your GitLab Server.
+
+1. Visit the Applications page in the Admin Area of your GitLab Server instance. Example: `https://gitlab.example.com/admin/applications` 
+2. Create an OAuth application with the following required settings.  
+   **Name:** Enter a name for your OAuth application.  
+   **Redirect URI:** Enter `https://app.probo.ci/auth/gitlab/callback` in this field.  
+   **Scopes:** API _(Check the API checkbox.)_
+3. Submit your changes to save the new OAuth application.
+
+Next you need to register your GitLab Server’s OAuth application with Probo.CI.
+
+## Register your GitLab Server with Probo.CI.
+
+1. Go to [http://app.probo.ci/#/self-hosted/new](http://app.probo.ci/#/self-hosted/new) to register a new self-hosted service.
+2. Enter the following required OAuth connection settings.
+
+{% note %}
+**Note:** The following required settings are found on your GitLab Server OAuth application settings page. Example: `https://gitlab.example.com/admin/applications/1`
+{% endnote %}
+
+- **Server URL:** The URL of your GitLab Server instance. Example: `https://gitlab.example.com`
+- **Client ID:** The `Application ID` string from your GitLab Server instance.
+- **Client Secret:** The `Secret` string from your GitLab Server instance.
+
+Now that you have your OAuth application configured and added to Probo.CI you can authenticate with your GitLab Server.
+
+## Authenticate your GitLab Server with Probo.CI.
+
+1. Visit [https://app.probo.ci/#/self-hosted](https://app.probo.ci/#/self-hosted).
+2. Enter your GitLab Server URL. Example: `https://gitlab.example.com`
+
+Congratulations, you should now be connected! Next you need to activate a Probo Organization.
+
+## Activate a Probo Organization.
+
+Once authenticated you need to activate your Probo Organization by choosing a subscription plan. New Probo.CI users are given a 2 month free trial to test things out and figure out what plan they want to use.
```

Comment:
Perhaps we should just link to the marketing site's Pricing page instead of mentioning the trial here? Just thinking about the future if we ever decide to change how the trial period works. We'd have to remember to come back and adjust this info.

Reply:
Fair enough.